modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/js/config.js

```diff
@@ -37,15 +37,6 @@
 							'plugin',
 						],
 					},
-					'liferay-editor-image-uploader': {
```

Comment:
This is technically a breaking change, but I doubt it was being used as an API... I'd say we can remove it, but maybe add an entry in the `BREAKING_CHANGES.md` file, just to be nice?

Reply:
On it!